cmd/podman/machine/rm.go

```diff
@@ -120,14 +120,12 @@ func rm(_ *cobra.Command, args []string) error {
 	// All actual removal of files and vms should occur after this
 	//
 
-	// TODO Should this be a hard error?
 	if err := providerRm(); err != nil {
 		logrus.Errorf("failed to remove virtual machine from provider for %q", vmName)
 	}
 
-	// TODO Should this be a hard error?
```

Comment:
Why are these removed? This almost certainly should be a hard error. Without this it will not exit >0 when it failed to remove the config.

Reply:
You're right, thanks for pointing that out. Silly mistake. Should be fixed.